src/elements/rich-link/RichlinkForm.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { FieldLayoutVertical } from "../../editorial-source-components/VerticalFieldLayout";
+import type { FieldValidationErrors } from "../../plugin/elementSpec";
+import type { FieldNameToValueMap } from "../../plugin/helpers/fieldView";
+import type { FieldNameToField } from "../../plugin/types/Element";
+import { CustomDropdownView } from "../../renderers/react/customFieldViewComponents/CustomDropdownView";
+import type { richlinkFields } from "./RichlinkSpec";
+
+type Props = {
+  fieldValues: FieldNameToValueMap<typeof richlinkFields>;
+  errors: FieldValidationErrors;
+  fields: FieldNameToField<typeof richlinkFields>;
+};
+
+export const RichlinkElementTestId = "RichlinkElement";
```

Comment:
I can't see that we use this element in our integration tests – if we don't test this element anywhere, we shouldn't need property.

Reply:
@jonathonherbert thanks well spotted, perhaps another PR to address adding an integration test for this element would be appropriate.

Reply:
Yes, let's not worry about this in this PR 👍 We probably need a better a story for integration testing our elements, which at the moment would be quite painful.